plugins/generator-1.20.1/forge-1.20.1/templates/tool.java.ftl

```diff
@@ -165,7 +165,7 @@ public class ${name}Item extends Item {
 	}
 
 	@Override public float getDestroySpeed(ItemStack itemstack, BlockState blockstate) {
-		return ${containsAnyOfBlocks(data.blocksAffected "blockstate")} ? ${data.efficiency}f : 1;
+		return <#if data.blocksAffected?has_content>${containsAnyOfBlocks(data.blocksAffected "blockstate")} ? ${data.efficiency} : </#if>1f;

```

Comment:
Why is this needed now and not needed with previous generators?

Reply:
I thought I could do this minor optimization to avoid 'obvious ternary operator' constructions like `false ? 4f : 1` that 1.19.4 would have generated.

Reply:
Makes sense

Reply:
and this introduced a bug: https://mcreator.net/forum/100049/error-code